drivers: sensors: sm351lt: Add new driver (#27433)
zephyrproject-rtos/zephyr: samples/sensor/sm351lt/README.rst

.. _sm351lt:

SM351LT: Magnetoresistive Sensor Example
########################################

Overview
********

This sample application periodically polls an SM351LT magnetoresistive sensor

> and displays if a magnet near to the sensor or not, via the console.

Reviewer: ```suggestion
and displays if a magnet is near to the sensor, via the console.
```
